plugin-gradle/README.md

```diff
@@ -139,10 +139,10 @@ To apply freshmark to all of the `.md` files in your project, with all of your p
 ```gradle
 spotless {
 	freshmark {
-		target 'README.md', 'CONTRIBUTING.md'	// defaults to '**/*.md'
-		propertiesFile('gradle.properties')		// loads all the properties in the given file
+		target 'README.md', 'CONTRIBUTING.md'			// defaults to '**/*.md'
+		propertiesFile 'fresh.xml', 'mark.properties'	// loads all the properties in the given files
 		properties {
-			it.put('key', 'value')				// specify other properties manually
+			it.put('key', 'value')						// specify other properties manually
```

Comment:
Still not sure whether we have a miss-understanding.
In confirm that in the live edit, there is no distinction between tabs and spaces. So I really got paranoid, downloaded ZIP of the master, and checked the plugin-gradle/README.md with VIM.
Everything in that file, which is within a code-block ('''), uses tabs. Everything outside code-blocks is using spaces.
I am using Markdown Preview Plus to play with Markdown. And at least that interpreter does not use mono-spaced fonts in there code-block. So I also do not see a good alternative to tabs within code-blocks.